xml/System.Reflection.Metadata/MethodBodyBlock.xml

```diff
@@ -51,9 +64,9 @@
         <Parameter Name="reader" Type="System.Reflection.Metadata.BlobReader" />
       </Parameters>
       <Docs>
-        <param name="reader">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="reader">The blob reader to read the method body.</param>
+        <summary>Creates a new instance of the <see cref="T:System.Reflection.Metadata.MethodBodyBlock" /> class using the specified blob reader.</summary>
+        <returns>A new instance of the <see cref="T:System.Reflection.Metadata.MethodBodyBlock" /> class.</returns>
         <remarks>To be added.</remarks>
```

Comment:
NIT: normally we add the Exceptions that could be thrown, seems the method could throw  `BadImageFormatException`